lib/firmware_model.dart

```diff
@@ -77,8 +77,14 @@ class FirmwareModel extends SafeChangeNotifier {
   Future<Map<String, List<FwupdRelease>>> _fetchReleases(
       List<FwupdDevice> devices) async {
     Future<List<FwupdRelease>> fetchReleases(FwupdDevice device) {
-      return _service.getReleases(device.id).catchError((_) => <FwupdRelease>[],
-          test: (e) => e is FwupdNothingToDoException);
+      return _service.getReleases(device.id).catchError(
+        (e) {
+          log.debug('could not fetch releases for device ${device.id}: $e');
```

Comment:
this turns out to be pretty noisy. :) do you think it's useful to have or could we silence it?

```
$ flutter test integration_test/
00:00 +0: loading /home/jpnurmi/Projects/canonical/firmware-updater/integration_test/firmware_updater_test.dart                                                                             B00:08 +0: loading /home/jpnurmi/Projects/canonical/firmware-updater/integration_test/firmware_updater_test.dart                                                                             
00:10 +0: fake webcam upgrade                                                                                                                                                               
DEBUG fwupd_notifier: 9 devices
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
WARNING: Multiple "Show Updates" buttons. Assuming the first.
DEBUG fwupd_service: install FwupdRelease(appstreamId: fakedevice.firmware, checksum: fc0aabcf98bf3546c91270f2941f0acd0395dd79, created: 2017-06-15 04:00:00.000Z, description: '<p>Fixes another bug with the flux capacitor to prevent time going backwards.</p>', filename: 2b8546ba805ad10bf8a2e5ad539d53f303812ba5, homepage: http://www.acme.com/, license: GPL-2.0+, locations: [./fakedevice124.cab], name: 'FakeDevice Firmware', protocol: null, remoteId: fwupd-tests, size: 17, flags: {FwupdReleaseFlag.isUpgrade}, urgency: FwupdReleaseUrgency.medium, uri: ./fakedevice124.cab, vendor: 'ACME Corp', version: '1.2.4')
DEBUG fwupd_service: on FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.updatable, FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 14:26:35.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.3, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: FwupdStatus.decompressing
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG fwupd_notifier: FwupdStatus.waitingForAuth
DEBUG fwupd_notifier: 0%
DEBUG fwupd_notifier: updated FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:00:56.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.3, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: FwupdStatus.deviceBusy
DEBUG fwupd_notifier: updated FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage, FwupdDeviceFlag.updatableHidden}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:00:56.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.3, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: FwupdStatus.deviceWrite
DEBUG fwupd_notifier: 1%
DEBUG fwupd_notifier: FwupdStatus.deviceRestart
DEBUG fwupd_notifier: FwupdStatus.deviceWrite
DEBUG fwupd_notifier: 2%
DEBUG fwupd_notifier: FwupdStatus.deviceWrite
DEBUG fwupd_notifier: FwupdStatus.deviceVerify
DEBUG fwupd_notifier: FwupdStatus.deviceVerify
DEBUG fwupd_notifier: updated FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage, FwupdDeviceFlag.updatableHidden}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:00:56.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.4, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: updated FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:00:56.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.4, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: FwupdStatus.deviceRestart
DEBUG fwupd_notifier: updated FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.updatable, FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:00:56.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.4, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: 95%
DEBUG fwupd_notifier: FwupdStatus.deviceBusy
DEBUG fwupd_notifier: 99%
DEBUG fwupd_notifier: 100%
DEBUG fwupd_notifier: 100%
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG fwupd_notifier: FwupdStatus.idle
DEBUG fwupd_notifier: FwupdStatus.idle
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
00:13 +1: fake webcam upgrade                                                                                                                                                               
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG fwupd_notifier: 9 devices
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG fwupd_service: install FwupdRelease(appstreamId: fakedevice.firmware, checksum: fc0aabcf98bf3546c91270f2941f0acd0395dd79, created: 2017-06-15 04:00:00.000Z, description: '<p>Fixes another bug with the flux capacitor to prevent time going backwards.</p>', filename: 2b8546ba805ad10bf8a2e5ad539d53f303812ba5, homepage: http://www.acme.com/, license: GPL-2.0+, locations: [./fakedevice124.cab], name: 'FakeDevice Firmware', protocol: null, remoteId: fwupd-tests, size: 17, flags: {}, urgency: FwupdReleaseUrgency.medium, uri: ./fakedevice124.cab, vendor: 'ACME Corp', version: '1.2.4')
DEBUG fwupd_service: on FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.updatable, FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:00:56.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.4, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: FwupdStatus.decompressing
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG fwupd_notifier: FwupdStatus.waitingForAuth
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG fwupd_notifier: 0%
DEBUG fwupd_notifier: updated FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:01:00.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.4, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: FwupdStatus.deviceBusy
DEBUG fwupd_notifier: updated FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage, FwupdDeviceFlag.updatableHidden}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:01:00.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.4, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: FwupdStatus.deviceWrite
DEBUG fwupd_notifier: 1%
DEBUG fwupd_notifier: FwupdStatus.deviceWrite
DEBUG fwupd_notifier: FwupdStatus.deviceWrite
DEBUG fwupd_notifier: 2%
DEBUG fwupd_notifier: FwupdStatus.deviceWrite
DEBUG fwupd_notifier: FwupdStatus.deviceWrite
DEBUG fwupd_notifier: FwupdStatus.deviceVerify
DEBUG fwupd_notifier: updated FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:01:00.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.4, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: FwupdStatus.deviceRestart
DEBUG fwupd_notifier: updated FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.updatable, FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:01:00.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.4, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: 95%
DEBUG fwupd_notifier: FwupdStatus.deviceBusy
DEBUG fwupd_notifier: 99%
DEBUG fwupd_notifier: 100%
DEBUG fwupd_notifier: 100%
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG fwupd_notifier: FwupdStatus.idle
DEBUG fwupd_notifier: FwupdStatus.idle
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
00:16 +2: fake webcam upgrade                                                                                                                                                               
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG fwupd_notifier: 9 devices
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG fwupd_service: install FwupdRelease(appstreamId: fakedevice.firmware, checksum: bc3c32f42cf33fe5aade64f999417251fd8208d3, created: 2017-06-15 04:00:00.000Z, description: '<p>Fixes a bug with the flux capacitor to avoid year 2038 overflow.</p>', filename: 7998cd212721e068b2411135e1f90d0ad436d730, homepage: http://www.acme.com/, license: GPL-2.0+, locations: [./fakedevice123.cab], name: 'FakeDevice Firmware', protocol: null, remoteId: fwupd-tests, size: 17, flags: {FwupdReleaseFlag.isDowngrade}, urgency: FwupdReleaseUrgency.medium, uri: ./fakedevice123.cab, vendor: 'ACME Corp', version: '1.2.3')
DEBUG fwupd_service: on FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.updatable, FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:01:00.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.4, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: FwupdStatus.decompressing
DEBUG fwupd_notifier: FwupdStatus.waitingForAuth
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG fwupd_notifier: 0%
DEBUG fwupd_notifier: updated FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:01:07.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.4, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: FwupdStatus.deviceBusy
DEBUG fwupd_notifier: updated FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage, FwupdDeviceFlag.updatableHidden}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:01:07.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.4, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: FwupdStatus.deviceWrite
DEBUG fwupd_notifier: 1%
DEBUG fwupd_notifier: FwupdStatus.deviceWrite
DEBUG fwupd_notifier: FwupdStatus.deviceWrite
DEBUG fwupd_notifier: 2%
DEBUG fwupd_notifier: FwupdStatus.deviceWrite
DEBUG fwupd_notifier: FwupdStatus.deviceWrite
DEBUG fwupd_notifier: FwupdStatus.deviceVerify
DEBUG fwupd_notifier: updated FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage, FwupdDeviceFlag.updatableHidden}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:01:07.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.3, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: FwupdStatus.deviceRestart
DEBUG fwupd_notifier: updated FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:01:07.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.3, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: 95%
DEBUG fwupd_notifier: updated FwupdDevice(checksum: null, created: 2022-09-14 10:42:19.000Z, deviceId: 08d460be0f1f9f128413f816022a6439e0078018, flags: {FwupdDeviceFlag.updatable, FwupdDeviceFlag.requireAc, FwupdDeviceFlag.supported, FwupdDeviceFlag.registered, FwupdDeviceFlag.canVerify, FwupdDeviceFlag.canVerifyImage}, guid: [b585990a-003e-5270-89d5-3705a17f9a43], icon: [preferences-desktop-keyboard], modified: 2022-09-15 15:01:07.000Z, name: 'Integrated Webcam™', parentDeviceId: null, plugin: test, protocol: com.acme.test, summary: Fake webcam, updateState: FwupdUpdateState.success, vendor: ACME Corp., vendorId: USB:0x046D, version: 1.2.3, versionBootloader: 0.1.2, versionFormat: FwupdVersionFormat.triplet, versionLowest: 1.2.0)
DEBUG fwupd_notifier: FwupdStatus.deviceBusy
DEBUG fwupd_notifier: 99%
DEBUG fwupd_notifier: 100%
DEBUG fwupd_notifier: 100%
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG fwupd_notifier: FwupdStatus.idle
DEBUG fwupd_notifier: FwupdStatus.idle
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device a3487e128cf1413519bce8e9a1ab3f5981e61458: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device cc9c1b47c3126a46379087db3bf18fa324f317ae: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 349bb341230b1a86e5effe7dfe4337e1590227bd: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device c430a03ca2a65dfe2412ff950c79c51f6aec1317: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 2292ae5236790b47884e37cf162dcf23bfcd1c60: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device d432baa2162a32c1554ef24bd8281953b9d07c11: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
DEBUG firmware_model: could not fetch releases for device 362301da643102b9f38477387e2193e57abaa590: FwupdNothingToDoException: No releases found
00:24 +3: All tests passed!   
```

Reply:
It was useful to find the bug, but has probably lost its value now..